src/impl/unicast/transport/NServiceBus.Unicast.Transport.Transactional.Config/Bootstrapper.cs

```diff
@@ -51,5 +51,6 @@ public void Init()
         public static IsolationLevel IsolationLevel { get; set; }
         public static TimeSpan TransactionTimeout { get; set; }
         public static bool SupressDTC { get; set; }
+        public static bool SuppressOutdatedTimeoutDispatchWarning { get; set; }

```

Comment:
you mean in AdvancedTransactionalConfig? 
all other settings are in bootstrapper class, so I didn't want to make an exception for this one and make people think why this one is different.